packages/combobox/src/LionCombobox.js

```diff
@@ -450,6 +460,17 @@ export class LionCombobox extends OverlayMixin(LionListbox) {
     return true;
   }
 
+  /**
+   * Return the value to be used for the input value
+   * @overridable
+   * @param {LionOption} option
+   * @returns {string}
+   */
+  // eslint-disable-next-line class-methods-use-this
+  _getTextboxValueFromOption(option) {
```

Comment:
Can we add some tests for this method and the expected behaviors when you override this method?

Reply:
done